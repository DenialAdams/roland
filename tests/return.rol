proc main() {
   print(get_a_cool_string());
}

func get_a_cool_string() -> String {
   return "The time to hesitate is through";
}
