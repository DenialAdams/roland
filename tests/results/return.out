The time to hesitate is through
